### PR TITLE
Allow more configuration for windows worker

### PR DESCRIPTION
**Pull request by sunjayBhatia**
- use concourse-windows-worker-release from bosh.io and add latest
  version to versions.yml
- add opsfile for ephemeral disk configuration
- add vm type configuration

### CODE REVIEW

File: cluster/operations/windows-worker.yml [hunk 1]
```diff
@@ -2,21 +2,23 @@
...
+    version: ((concourse_windows_worker_version))
+    sha1: ((concourse_windows_worker_sha1))
+    url: https://bosh.io/d/github.com/pivotal-cf-experimental/concourse-windows-worker-release?v=((concourse_windows_worker_version))
 
 - type: replace
   path: /stemcells/-
   value:
     alias: windows
-    os: windows2012R2
+    os: ((windows_worker_os_version))
```
Comment (vito) [inline cluster/operations/windows-worker.yml]: This should probably be just `windows_worker_os`

Out of curiosity, what other value did you need to specify here?

**Review by vito — CHANGES_REQUESTED** [review]

File: cluster/operations/windows-worker.yml (hunk 1 above)
Comment (sesmith177) [reply cluster/operations/windows-worker.yml]: We (garden windows team) use windows2016 workers in our concourse pipeline

**Comment by aminjam** [comment]
@vito @jama-pivotal We were wondering what y'all think about this PR? We are running a forked version of the concourse-deployment to have this configuration and would prefer to use the upstream instead.

**Comment by sesmith177** [comment]
@vito is there anything more we need to do to help this get merged?

**Review by vito — APPROVED** [review]

**Comment by vito** [comment]
Thanks! Sorry for the delay, I've been on vacation.

**Pull request merged**